### ISSUE TITLE
Feature request: Allow to customize key for previous position register

### ISSUE THREAD

**Issue by tobimensch** [philc/vimium#2263]
Currently single quote and backtick are hard coded in:
https://github.com/philc/vimium/blob/master/content_scripts/marks.coffee

I want to bind the goto mark feature to q, and have to previous position register also on q.
(Since q is way more convenient to use on my keyboard layout compared to backtick)

What I want is the possiblity to set a custom key for previous position register in the options.

**Comment by smblott-github** [comment]
In vim, the backtick and quote are hardwired to this too (I think).

When this was implemented, I wanted it to be that the *previous-position register* was named after the last character of the activating key sequence.  So, with

    map gm gotoMark

`gmm` would jump to the previous position.  If I understand correctly, this is more or less the same as your proposal.

There was loud objection to the idea, so we restricted the feature to backtick and quote.  There's some merit to that decision.  `gmm` should mean *goto mark `m`*.  Also, it has no vim analog and is undiscoverable.

**Comment by tobimensch** [comment]
@smblott-github:

I don't think you really got the problem. backtick and single quote are both very inconvenient on a German keyboard. So I need a way to use some other key instead of backtick. I believe in vim you can remap such key bindings. But in vimium it isn't possible in this case, because the single quote and backtick characters are hard coded.
